algorithms/searching.md

```diff
@@ -24,14 +24,8 @@ Searching for an item in an array is a common need. Software developers should k
 - [Linear search video](https://www.youtube.com/watch?v=vZWfKBdSgXI)
 - [Binary search video](https://www.youtube.com/watch?v=5xlIPT1FRcA)
 - [Depth-First and Breadth-First Searches](https://youtu.be/zaBhtODEL0w)
-
-Additional resources:
-- [Linear and binary search](https://medium.com/@connorleech/implement-linear-and-binary-search-algorithms-with-javascript-2149997588f0)
-- [In-depth articles about binary search](https://www.khanacademy.org/computing/computer-science/algorithms/binary-search/a/binary-search)
-- [More about log(n)](https://hackernoon.com/what-does-the-time-complexity-o-log-n-actually-mean-45f94bb5bfbf)
-- [Linear + binary search video](https://www.youtube.com/watch?v=wNVCJj642n4)
-- [Binary search + code implementation video](https://www.youtube.com/watch?v=P3YID7liBug)
-
+- [Searching algorithm (slides)](https://docs.google.com/presentation/d/1x8xO_URwZVoB4JxTNN90z0sHSxoYEq0INOH_IIi8SLE/edit#slide=id.p)
+- [Searching algorithm (video walkthrough of slides)](https://drive.google.com/open?id=1iHgUlPK2G9Gk3mbdOSObUKOFld_mALrs)
```

Comment:
I think if this video is uploaded on YouTube and set as unlisted then it would be good. Because on Google Drive people can only play a video some particular number of times only. @alodahl What your opinion ?

Reply:
Particular number of times? I don’t understand. It should be viewable by anyone.

Reply:
If it should be view-able to anyone then it could be made public. 
By particular number of times I mean that many times I have heard from people that if many number of people are watching the same video on GDrive then the people coming next to watch will get some error.

Reply:
I’ll let the director know. How many viewers are we talking? FYI, we have a library of about 30 videos that are in google drive, so if we were to change it would be a big task. For this particular PR I say we don’t change it.

Reply:
I don't know about the number of viewers but you will get this error message,
**"Too many users have viewed or downloaded this file recently. Please try accessing the file again later. If the file you are trying to access is particularly large or is shared with many people, it may take up to 24 hours to be able to view or download the file. If you still can't access a file after 24 hours, contact your domain administrator."**

Reply:
Good to know, thanks.

Reply:
Most welcome.

Reply:
I know that 30 videos will take time but it would be good for future.